tests/ExamplesTest.php

```diff
@@ -179,6 +182,39 @@ public static function provideExamples(): Generator
         ];
     }
 
+    /**
+     * MongoDB Atlas Search example requires a MongoDB Atlas M10+ cluster with MongoDB 7.0+ and sample data loaded.
+     * Tips for insiders: if using a cloud-dev server, append ".mongodb.net" to the MONGODB_URI.
+     *
+     * @group atlas
+     */
+    public function testAtlasSearch(): void
+    {
+        $uri = getenv('MONGODB_URI') ?? '';
+        if (! self::isAtlas($uri)) {
+            $this->markTestSkipped('Atlas Search examples are only supported on MongoDB Atlas');
+        }
+
+        $this->skipIfServerVersion('<', '7.0', 'Atlas Search examples require MongoDB 7.0 or later');
+
+        $client = new Client($uri);
+        $collection = $client->selectCollection('sample_airbnb', 'listingsAndReviews');
+        $count = $collection->estimatedDocumentCount();
+        if ($count === 0) {
+            $this->markTestSkipped('Atlas Search examples require the sample_airbnb database with the listingsAndReviews collection');
```

Comment:
Do I assume correctly that this is for the benefit of running the test locally, and that it will most likely be skipped in CI because we don't load sample data?

I'm OK either way, but just wanted to ask.

Reply:
We added the `@group atlas`, but I think I can insert some data to unit the collection in tests if it does not exist. We would benefit from having the test running in CI.